GenderPayGap.WebUI/Controllers/Account/CloseAccountNewController.cs

```diff
@@ -14,11 +12,13 @@
 using GenderPayGap.WebUI.Services;
 using GovUkDesignSystem;
 using GovUkDesignSystem.Parsers;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 
 namespace GenderPayGap.WebUI.Controllers.Account
 {
     [Route("manage-account")]
+    [Authorize(Roles = LoginRoles.GpgEmployer)]

```

Comment:
This line is great.
But, I've just realised that it might break the `close/completed` view (because the user will be logged out at that stage).
We should test that (and add `[AllowAnonymous]` on the `close/completed` action if needed)